project/src/main/java/com/google/sps/classes/Event.java

```diff
@@ -32,8 +32,11 @@
   private final boolean isMine;
   private final String entityType;
   private final String imageKey;
+  private final int day;
+  private final int numAttendees;
+  private final int maxCapacity;
  
-  public Event(long id, String name, String location, String date, String time, String description, String attendance, String type, long timeStamp, boolean isMine, String entityType, String imageKey) {
+  public Event(long id, String name, String location, String date, String time, String description, String attendance, String type, long timeStamp, boolean isMine, String entityType, String imageKey, int day, int numAttendees, int maxCapacity) {
```

Comment:
Could you make this line not too long. Usually we have threshold of 80 chars per line

Reply:
done!